frontend/packages/container-security/src/plugin.ts

```diff
@@ -96,7 +97,7 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'Dashboards/Overview/Health/Resource',
     properties: {
-      title: 'Image Vulnerabilities',
+      title: i18n.t('container-security~Image Vulnerabilities'),
```

Comment:
avoid use of global use of i18n, can leave `plugin.ts` for now

Reply:
done